plugins/drivers/proto/driver.proto

```diff
@@ -495,6 +495,9 @@ message TaskConfig {
 
     // NodeId is the ID of the node where the associated allocation is running
     string node_id = 21;
+
+    // ParentJobID is the parent id for dispatch and periodic jobs
+    string parent_job_id = 22;
```

Comment:
`plugins/drivers/proto/driver.pb.go` is now out of sync and needs to be regenerated as well. `make proto` will do so.